client/components/empty-content/index.jsx

```diff
@@ -22,7 +22,7 @@ class EmptyContent extends Component {
 		title: PropTypes.oneOfType( [ PropTypes.string, PropTypes.array ] ),
 		illustration: PropTypes.string,
 		illustrationWidth: PropTypes.number,
-		line: PropTypes.oneOfType( [ PropTypes.string, PropTypes.array ] ),
+		line: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element, PropTypes.array ] ),
```

Comment:
This can just be a `PropTypes.node`. Addressing in #54790 together with 20+ other components where this prop type is also unnecessarily complicated.